src/Sylius/Bundle/TaxonomyBundle/Resources/translations/validators.en.yml

```diff
@@ -21,3 +21,4 @@ sylius:
         locale:
             not_blank: Please enter the locale.
             invalid: This value is not a valid locale.
+            unique: Translation in {{ value }} locale code already exists.
```

Comment:
```suggestion
            unique: Translation for the "{{ value }}" locale code already exists.
```

Reply:
Final version `A translation for the {{ value }} locale code already exists.`